app/protocol/flows/handshake/receiveversion.go

```diff
@@ -18,6 +19,8 @@ var (
 	// minAcceptableProtocolVersion is the lowest protocol version that a
 	// connected peer may support.
 	minAcceptableProtocolVersion = uint32(3)
+
+	defaultMaxProtocolVersion = uint32(4)
```

Comment:
The word "default" here makes it sound like the max protocol may change during runtime. Why not maxAcceptableProtocolVersion or defaultProtocolVersion?